task01-html-css/characters.css

```diff
@@ -0,0 +1,71 @@
+/**
+ * All Rights Reserved
+ * This software is property information of AKUREY S.A.
+ * Contact us at contact@akurey.com
+ * @summary short description for the file
+ */
+
+.grid-container {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    gap: 20px;
+
+    width: 940px;
+    height: 279.65px;
+    overflow-x: scroll;
+
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+}
+
+.grid-item {
+    position: relative;
+    /* display: flex;
+    align-items: flex-end;
+    text-align: center; */
+
+    width: 219.74px;
+    height: 279px;
+
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+}
+
+.text-item{
+    position: absolute;
+    width: 218.61px;
+    height: 38.65px;
+    
+    top: 211.36px;
+
+    font-family: 'Roboto Slab';
+    font-style: normal;
+    font-weight: 700;
+    font-size: 22px;
+    line-height: 28px;
+
+    display: flex;
+    align-items: flex-end;
+    text-align: center;
+
+    color: #FFF;
+    z-index: 15;
+}
+
+.filter-item{
+    position: absolute;
+
+    width: 219.74px;
+    height: 279px;
+    background: linear-gradient(179.8deg, rgba(255, 255, 255, 0) 50.33%, #000000 97.62%);
+    mix-blend-mode: multiply;
+    border-radius: 19.7522px;
+    z-index: 10;
+}
+
+.image-item{
+    position: absolute;
```

Comment:
Instead of using absolute, try this:
`    width: 100%;
    height: 100%;
    object-fit: cover;`
    
It will adjust the parent box height and width.